_sass/minima/_layout.scss

```diff
@@ -51,6 +55,14 @@
     border-radius: 5px;
     text-align: right;
 
+    label[for="nav-trigger"] {
+      display: block;
+      float: right;
+      width: 36px;
+      height: 36px;
+      z-index: 2;
```

Comment:
@Ambrosiani I'd wondered about that, but it'll only make sense if the hamburger gets used for desktop. `cursor: pointer;` doesn't seem to make much sense for me if it's in "touch" ?